### PR TITLE
google-spreadsheet: add includeCells parameter in GoogleSpreadsheet.loadInfo function

### PR DESCRIPTION
**Pull request by pavellishin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [<<url here>>](https://github.com/theoephraim/node-google-spreadsheet/blob/07b0783/lib/GoogleSpreadsheet.js#L261)
  - This arg is not in the docs, but it is in the code.
- ~[ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.~ not applicable.

### CODE REVIEW

File: types/google-spreadsheet/index.d.ts [hunk 1]
```diff
@@ -1168,8 +1168,9 @@ export class GoogleSpreadsheet implements SpreadsheetBasicProperties {
     /**
      * @description
      * load basic Spreadsheet document properties and child worksheets
+     * Pass in (optional) truthy parameter to also load grid data.
```
Comment (EndBug) [inline types/google-spreadsheet/index.d.ts:1171]: I'd use JSDoc to document the parameter

```suggestion
     * 
     * @param includeCells whether to also load grid data
```

**Review by EndBug — CHANGES_REQUESTED** [review]
Nice catch!

This parameter is not documented in the official docs, but it's present in the source code, [here](https://github.com/theoephraim/node-google-spreadsheet/blob/456ce2563db894ed9db541c26dc415f7bfde01b6/lib/GoogleSpreadsheet.js#L260-L270).
I've opened an issue to track the discussion regarding the docs: https://github.com/theoephraim/node-google-spreadsheet/issues/622

Regarding this PR:
- I've made a suggestion to use JSDoc for the parameter docs
- I'd also add a couple of lines to the tests, so that we can merge this autonomously. You can just add something like this below [this line](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e62c6f70096387d5cbcdf7a8e49aff8f60a753fb/types/google-spreadsheet/google-spreadsheet-tests.ts#LL26C70-L26C70):

```ts
    await doc.loadInfo();
    await doc.loadInfo(false);
    await doc.loadInfo(true);
```

**Comment by pavellishin** [comment]
Thanks for the feedback, just pushed up the suggested changes!

**Review by EndBug — APPROVED** [review]
LGTM ✅ 

Thanks for the PR 💖

**Comment by pavellishin** [comment]
Ready to merge

**Pull request merged**